sklearn/ensemble/voting_classifier.py

```diff
@@ -119,11 +162,22 @@ def fit(self, X, y):
         self.le_ = LabelEncoder()
         self.le_.fit(y)
         self.classes_ = self.le_.classes_
-        self.estimators_ = []
 
-        for name, clf in self.estimators:
-            fitted_clf = clone(clf).fit(X, self.le_.transform(y))
-            self.estimators_.append(fitted_clf)
+        cloned = [clone(clf) for _, clf in self.estimators]
+        transformed_y = self.le_.transform(y)
+
+        n_jobs, _, starts = _partition_estimators(len(self.estimators),
+                                                  self.n_jobs)
+        self.estimators_ = list(itertools.chain.from_iterable(
+            Parallel(n_jobs=self.n_jobs, backend=self.backend)(
+                delayed(_parallel_fit_estimators)(
+                    cloned[starts[i]:starts[i + 1]],
+                    X,
+                    transformed_y,
+                    sample_weight
+                ) for i in range(n_jobs)
```

Comment:
If there is some reason behind this, then we would have to do wherever `n_splits > n_jobs` in cross-validation strategies and in 20 other places!